src/main/webapp/queue.js

```diff
@@ -15,6 +16,25 @@ db.collection('rooms') // eslint-disable-line no-undef
       getRoomQueue(roomParameters);
     });
 
+db.collection('rooms') // eslint-disable-line no-undef
+    .doc(roomParameters)
+    .collection('CurrentVideo')
+    .doc('PlaybackData')
+    .onSnapshot(function(snapshot) {
+      if (snapshot.get('videoId') == '') {
+        document.getElementById('skipBtn').disabled = true;
+      } else {
+        document.getElementById('skipBtn').disabled = false;
+      }
+      document.getElementById('skipCounter').innerHTML =
+          'Votes to skip video: ' + snapshot.get('skipVotes');
+      if (snapshot.get('skipVotes') >= snapshot.get('numPeopleWatching')/2 &&
```

Comment:
numPeopleWatching becomes 0 right as the video switches (which could trigger this accidentally), so I think it'd be safer to use the information from Caroline's guest collection to find the total number of people because it'll always just have the total number of people in the room.

Reply:
I'd like to keep it tied with the field instead to maintain consistency in the people who may be watching a video instead of those populating the lounge. I added extra protection for this case in that skipping is only prevented when numPeopleWatching is greater than 0.

Reply:
Alright, sounds good.